docs/en/users/03_Main_view.md

```diff
@@ -239,6 +239,8 @@ For example, you can enter multiple instances of `f:`, `author:`, `intitle:`, `i
 Combining several search criteria implies a logical *and*, but the keyword ` OR `
 can be used to combine several search criteria with a logical *or* instead: `author:Dupont OR author:Dupond`
 
+When combining multiple negative operators (e.g. `!intitle:'thing1' AND !intitle:'thing2'`), you'll need to consider [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws). See [this GitHub comment](https://github.com/FreshRSS/FreshRSS/issues/3236#issuecomment-891219460) for more details on the usage of `AND` vs `OR` when using combining multiple negative operators.
```

Comment:
Darn it. I just realized I typed "using combining" instead of picking one of those words only. 😅

Reply:
This sentence seems a bit vague to me. Who (besides @math-GH :-P) would know what De Morgan's law is? More of a "btw, this is because of…" than "you need to consider".

Reply:
Also valid a valid point. I'll submit another PR that rewords this a bit later. Perhaps "PRs when sleep-deprived" are not so strong. 🙃

Reply:
#3992